src/components/CustomizedPipeline/CustomizePipelinesModal.tsx

```diff
@@ -12,7 +12,7 @@ export const createCustomizeAllPipelinesModalLauncher = (
 ) =>
   createRawModalLauncher(CustomizeAllPipelines, {
     'data-testid': 'customized-all-pipelines-modal',
-    variant: ModalVariant.large,
+    width: '75%',
```

Comment:
Just a suggestion. Fixed width is still causing problems is smaller screens. I guess using <Truncate> in external link text it is being displayed properly.
```
<ExternalLink
                href={
                  component.spec.containerImage.startsWith('http')
                    ? component.spec.containerImage
                    : `https://${component.spec.containerImage}`
                }
                text={<Truncate content={component.spec.containerImage} />}
              />
              ```

Reply:
Issue still in smaller screen sizes
<img width="1187" alt="Screenshot 2024-04-18 at 5 47 59 PM" src="https://github.com/openshift/hac-dev/assets/24852534/05b5c3ad-a1e8-426a-ada2-70ab12368118">

Using Truncate on External link text
<img width="1108" alt="Screenshot 2024-04-18 at 5 56 14 PM" src="https://github.com/openshift/hac-dev/assets/24852534/31e82376-f5cd-4037-8f92-023f395ad85e">

Reply:
Fixed it with better handling of table row width - 

https://github.com/openshift/hac-dev/assets/6041994/195d03ba-345b-4562-a866-9fd12c3be1a3